README.md

```diff
@@ -30,6 +30,9 @@ The first command will build the source of your application. The second command
 - **Stack Name**: The name of the stack to deploy to CloudFormation. This should be unique to your account and region, and a good starting point would be something matching your project name.
 - **AWS Region**: The AWS region you want to deploy your app to.
 - **EnableWebhooks**: Specify whether you want the solution deployed with Webhooks.
+- **ExistingVpc**: Specify an existing VPC Id, leave blank to have one created.
```

Comment:
If I'm bringing my own VPC it would be useful to know what requirements this app has. Something like https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/prerequisites.html#verify-your-vpc-configuration

Reply:
Good call, i'll create/add one.

Reply:
Added a section to the pre-reqs at the top of this readme around using an existing VPC.